tools/doc/html.js

```diff
@@ -36,13 +41,31 @@ marked.setOptions({ renderer });
 
 const docPath = path.resolve(__dirname, '..', '..', 'doc');
 
+// add class attributes to index navigation links
```

Comment:
We usually use first uppercased letter and period in the end in comments lately)
```js
// Add class attributes to index navigation links.
```